app/components/date-time-picker.hbs

```diff
@@ -1,14 +1,25 @@
 <div class="au-o-grid au-o-grid--small">
   <div class="au-o-grid__item au-u-2-6">
-    <AuLabel>Datum</AuLabel>
-    <WuDatepicker class="au-c-datepicker" @onChange={{this.onChangeDate}} @value={{this.date}}/>
+    {{#let (concat this.uniqueId "-date") as |id|}}
+      <AuLabel for={{id}}>Datum</AuLabel>
+      <AuDatePicker
```

Comment:
Another thing I noticed is that typing in dates doesn't seems to work yet, while it does work in the <duet-date-picker> demo. Clearing the date input with backspace also throws an exception in the console since that isn't handled yet by AuDatePicker.

Reply:
since this also did not work correctly in the old component, I'm not going to see that as blocking for now

Reply:
ah doing some more testing, it is a bit worse.

Reply:
Should be ok now.